spec/system/accessibility_spec.rb

```diff
@@ -111,36 +111,4 @@
       expect(page).to be_axe_clean.skipping("aria-allowed-attr")
     end
   end
-
-  describe "Print results page" do
```

Comment:
deleted this spec as the print page is no longer a thing. I did update this to run through the code in download.html.slim but in running through this check, it caused the PDF to actually download and get added as a new file, and couldn't work out how to stop it from doing that!